Jenkinsfile

```diff
@@ -0,0 +1,12 @@
+wrappedNode(label: 'linux && x86_64', cleanWorkspace: true) {
```

Comment:
We discussed offline and since we run tests against a dind image the host version doesn't matter too much.  I think we can keep this label so that we have a larger pool of hosts which should help reduce the time a job is waiting in a queue.